component/src/test/resources/testng.xml

```diff
@@ -27,6 +27,7 @@
             <class name="io.siddhi.extension.io.file.FileSourceLineModeTestCase"/>
             <class name="io.siddhi.extension.io.file.FileSourceBinaryModeTestCase"/>
             <class name="io.siddhi.extension.io.file.FileSinkTestCase"/>
+            <class name="io.siddhi.extension.io.file.FileFunctionsTestCase"/>
         </classes>
     </test>
 </suite>
```

Comment:
Missing new line

Reply:
@grainier fixed in https://github.com/siddhi-io/siddhi-io-file/commit/c2d3273f60ae4d82310996901fd59f38bb4c7975